docs/courses/csintro3/orientation/math.md

```diff
@@ -1,101 +1,204 @@
 
-| blocks | JavaScript |
-| -- | -- |
-| x  |  * |
-| +  |  + | 
-| %  |  / | 
+## Concept: Math Operators
 
-etc
+In Blocks, each step of an equation needed it's own block. This can make formulas difficult to properly express, as it can be hard to identify (or change) the order in which they are evaluated.
 
-### Complex Expressions
+In JavaScript, the same formulas can be easier to express, as the syntax (structure) is much closer to what is used when evaluating math by hand or with a scientific calculator.
 
-```typescript
-let num: number = 5 + 3 * 2;
-console.log("" + num);
+| Operation         | Block                 | JavaScript            |
+| :---------------- | :-------------------: | :-------------------- |
+| Addition          | `[let x = (6 + 2)]`   | `let x = 6 + 2;`      |
+| Subtraction       | `[let x = (6 - 2)]`   | `let x = 6 - 2;`      |
+| Multiplication    | `[let x = (6 * 2)]`   | `let x = 6 * 2;`      |
+| Division          | `[let x = (6 / 2)]`   | `let x = 6 / 2;`      |
+
+## Example #1: Complex Expressions
+
+1. Review the examples below
+2. Identify what is different between the Blocks and the JavaScript in each pair
+3. Run the examples: identify what the end result will be (it may be useful to add ``game.splash`` or ``console.log`` to display the value)
+
+### Example #1a: Addition and Subtraction
+
+```blocks
+let num: number = 1 + 2 - 3;
 ```
 
 ```typescript
-let num: number = 15 / 3 * 8;
-console.log("" + num);
+let num: number = 1 + 2 - 3;
 ```
 
-## Assignment Operators
-
-### Show increment / change by transition (blocks -> JavaScript)
+### Example #1b: Addition and Multiplication
 
-```typescript
-let num: number = 0;
-num = num + 5;
-num = num + 3;
+```blocks
+let num: number = 5 + 3 * 2;
 ```
 
 ```typescript
-let num: number = 0;
-num += 5;
-num += 3;
+let num: number = 5 + 3 * 2;
 ```
 
-### Special case: add or subtract 1 (increment / decrement operators)
+### Example #1c: Division and Multiplication
 
-```typescript
-let num: number = 0;
-num += 1;
-num += 1;
-num += -1;
-num += 1;
+```blocks
+let num: number = 5 + 3 * 2;
 ```
 
 ```typescript
-let num: number = 0;
-num++;
-num++;
-num--;
-num++;
+let num: number = 24 / 3 * 4;
+```
+
+## Student Task #1: Creating Blocks
+
+1. Create a new project in @boardname@. Go to the JavaScript view
+2. Recreate the expression below in **JavaScript**
+3. To confirm the JavaScript expression is correct, switch to Blocks and check that the result is the same
+4. **Challenge:** change the ``+`` to a ``-``, and switch back to JavaScript. What has changed?
+
+```blocks
+let num: number = 15 + 8 / 3
 ```
 
-Explain why -> very commonly used, easier to write as a short hand, etc
+## Concept: Order of Operations
+
+JavaScipt uses a **PEMDAS** structure to determine the order in which operations are evaluated. This standards for **P**arentheses, **E**xponents, **M**ultiplication or **D**ivision, **A**ddition or **S**ubtraction.
```

Comment:
Huh, looks like gh flavored md is outdated with regards to bolding. Renders properly (as it does on [most any other md renderer](https://meta.stackexchange.com/questions/140706/how-do-i-make-part-of-a-word-bold-using-markdown))